packages/block-library/src/social-link/edit.js

```diff
@@ -52,7 +52,7 @@ const SocialLinkEdit = ( { attributes, setAttributes, isSelected } ) => {
 				className={ classes }
 				onClick={ () => setPopover( true ) }
 			>
-				<IconComponent />
+				<em />
```

Comment:
Semantically, I don't think an `em` is the right element to use here. Why not a `span`?

Reply:
Sure, let's move to a span. Em was perhaps a bit too clever.